jenkins-automation.groovy

```diff
@@ -0,0 +1,164 @@
+// ** templates **
+// ***************
+
+job('template-base') {
+  wrappers {
+    colorizeOutput()
+  }
+
+  // handle failure
+}
+
+
+job('template-base-js') {
```

Comment:
Is the intent to have templates for all of our stacks here? `template-base-python`, `template-base-qu`, etc.?  Won't that make this file kind of messy/big? Is it possible to break this up and import?  Also, will this work cleanly if we have two different version of python or will we need `template-base-python26` and `template-base-python27`?